reference/api/documents.md

```diff
@@ -236,10 +254,18 @@ Delete one document based on its unique id.
 #### Response: `202 Accepted`
 
 ```json
-{ "updateId": 1 }
+{
+    "uid": 1,
+    "indexUid": "movies",
+    "status": "enqueued",
+    "type": "documentsDeletion",
+    "enqueuedAt": "2021-08-11T09:25:53.000000Z"
+}
 ```
 
-This `updateId` allows you to [track the current update](/reference/api/updates.md).
+You can use this `uid` to get more details on [the status of the task](/reference/api/tasks.md#get-task-status-by-uid).
+
+This `uid` allows you to [track the current task](/reference/api/tasks.md).
```

Comment:
If we put this information here, we should also add this line on all previous and next examples for consistency. 😇

Reply:
I do have it for all the examples on this page. Do you mean all the responses under API reference?

Reply:
I'm talking about the `line 268` which during my review was only present in this example

Reply:
Sorry if my previous comment wasn’t clear 
I meant I have the line:
>This `uid` allows you to [track the current task](/reference/api/tasks.md).

for all the examples on the /reference/api/documents.md page. You can see that on lines: 145, 205, 235, 268, 306.

Do you want me to add it to all the responses that return a `uid`? E.g. after lines 104, 143, 171 on  /reference/api/indexes.md and other pages under API reference?

Reply:
Oh yes, I know what you mean! I don't know what's better actually. I think it's a way to make sure that we indicate that the writings operation are asynchronous.

Is it possible from your side to define reusable snippets to display the same block of information in several places in the documentation code and modify it by making a single modification in one single file?

Reply:
We don’t have any reusable snippets for now but I'll add this